### PR TITLE
Docs fixes in raster read 

### PR DESCRIPTION
**Pull request by vpipkt**
#250 #89

### CODE REVIEW

**Comment by vpipkt** [comment]
@courtney-whalen take a quick look please let us know if you think it handles #250 less already logged #252

**Comment by courtney-whalen** [comment]
@vpipkt just reviewed - looks good to me! I closed #250.

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 1]
```diff
@@ -56,19 +58,19 @@ RasterFrames relies on three different IO drivers, selected based on a combinati
...
+| `file://`           | yes         | yes      | no       |
+| `http://`           | yes         | yes      | no       |
+| `https://`          | yes         | yes      | no       |
+| `ftp://`            | `/vsicurl/` | yes      | no       |
+| `hdfs://`           | `/vsihdfs/` | no       | yes      |
+| `s3://`             | `/vsis3/`   | yes      | no       |
+| `s3n://`            | no          | no       | yes      |
+| `s3a://`            | no          | no       | yes      |
+| `wasb://`           | `/vsiaz/`   | no       | yes      |
+| `wasbs://`          | no          | no       | yes      |
```
Comment (metasim) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: 👍 FYI: The unicode characters caused problems with (LaTeX) PDF generation.

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 2]
```diff
@@ -142,15 +144,17 @@ spark.read.raster(uri, lazy_tiles=False) \
...
 
 ## Multiband Rasters
 
-A multiband raster represents a three dimensional numeric array. The first two dimensions are spatial, and the third dimension is typically designated as different bands. The bands could represent intensity of different wavelengths of light (or other electromagnetic radiation), or they could represent other phenomena such as measurement time, quality indications, or additional measurements.
+A multiband raster represents a three dimensional numeric array. The first two dimensions are spatial, and the third dimension is typically designated as different bands. The bands could represent intensity of different wavelengths of light (or other electromagnetic radiation), or they could represent other phenomena such as measurement time, quality indications, or additional measurements. 
+
+Multiband rasters files have a strictly ordered set of bands, which are typically indexed from 1. Some files have metadata tags associated with each band. Some files have a color interpetation metadata tag indicating how to interpret the bands.
```
Comment (metasim) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: > which are typically indexed from 1

Do we need to be careful here in indicating that "band index" specifications in RasterFrames are zero based?

Comment (metasim) [reply pyrasterframes/src/main/python/docs/raster-read.pymd]: I see that you point that out below, but I think further clarification that one-based is "colloquially" or "conventionally" in the context of EO dialog.

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 3]
```diff
@@ -180,6 +184,29 @@ mb2 = spark.read.raster(catalog=spark.createDataFrame(mb_cat),
...
+## GeoTrellis
+
+### GeoTrellis Catalogs
+
+[GeoTrellis][GeoTrellis] is one of the key libraries that RasterFrames builds upon. It provides a Scala language API to working with large raster data with Apache Spark. RasterFrames provides a DataSource that supports both reading and @ref:[writing](raster-write.md#geotrellis-layers) with GeoTrellis.
+
+A GeoTrellis catalog is a set of GeoTrellis layers. We can read a dataframe giving details of the content of a catalog using the following. The scheme is typically `hdfs` or a cloud storage provider like `s3` or `wasb`.
+
+```python, evaluate=False
+gt_cat = spark.read.format('geotrellis-catalog').load('scheme://path-to-gt-catalog')
```
Comment (metasim) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: We should create an extension method for this to be consistent with other usesages.

File: pyrasterframes/src/main/python/docs/raster-read.pymd (hunk 1 above)
Comment (vpipkt) [reply pyrasterframes/src/main/python/docs/raster-read.pymd]: yes and i think the use of `+` and `-` broke the table formatting in paradox or someplace

File: pyrasterframes/src/main/python/docs/raster-read.pymd (hunk 2 above)
Comment (vpipkt) [reply pyrasterframes/src/main/python/docs/raster-read.pymd]: My understanding of GDAL anyways is that they are defined as one-based index: 

https://gdal.org/doxygen/classGDALDataset.html#ad96adcf07f2979ad176e37a7f8638fb6 

My concern is the user that's a bit familiar with EO data does a `gdalinfo` and sees band 1 and 2 are what they need, then put `band_indexes=(1,2)`  


Maybe I'm missing your point?

**Pull request merged**